frontend/public/components/storage/attach-storage.tsx

```diff
@@ -85,7 +85,7 @@ const AttachStorage: React.FC<AttachStorageFormProps> = (props) => {
                 key={k}
                 isChecked={activeProvider === k}
                 onChange={handleChange}
-                label={v.name}
+                label={t(v.name)}
```

Comment:
I don't know that you can assume each storage provider is a translation key if it's passed in as an extension.

Reply:
I have changed the name of the parameter to `nameKey` similar to what is done in `HorizontalNav` that should make things clearer for extension consumers. WDYT?